### PR TITLE
Using SD object

### PR DESCRIPTION
**Pull request by Pulokesh**
1. Alpha and beta pool of orbitals are created using extracted coefficients of MO 
2. Used "excitation" object to create proper index of spin for the excitation.
3. Alpha and beta spin indices are then used to create SD basis list.
4. Overlaps are calculated using SD_overlap

Currently, only excitations from HOMO is considered.

### CODE REVIEW

File: src/spin_indx.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import sys
+import math
+
+if sys.platform=="cygwin":
+    from cyglibra_core import *
+elif sys.platform=="linux" or sys.platform=="linux2":
+    from liblibra_core import *
+
+
+def index_spin(params,active_space):
```
Comment (alexvakimov) [inline src/spin_indx.py]: add documentation

File: src/main.py [hunk 2]
```diff
@@ -59,12 +60,14 @@ def main(params):
     ntraj = nstates*ninit*num_SH_traj
 
     #######
-    #active_space = [6,7]  # For C2H4 
+    if params["interface"]=="QE":
+        active_space = [5,6,7]  # For C2H4
```
Comment (alexvakimov) [inline src/main.py]: please check my previous comments about this block (here and next few lines).

File: src/x_to_libra_qe.py [hunk 3]
```diff
@@ -83,34 +84,38 @@ def qe_to_libra(params, E, sd_basis, label, mol, suff, active_space):
...
     all_grads = [] # this will be a list of lists of VECTOR objects
     E2 = MATRIX(nstates,nstates)
 
 
     #======== Run QE calculations and get the info at time step t+dt ========
-
+    
     for ex_st in xrange(nstates): # for each excited configuration
                                   # run a separate set of QE calculations
+        idx = params["excitations"][ex_st]
```
Comment (alexvakimov) [inline src/x_to_libra_qe.py]: i'm not sure where "idx" is used - it looks like it is not used below

Comment (Pulokesh) [inline src/x_to_libra_qe.py]: idx.from_orbit[0] and idx.to_orbit[0] is used in the code. 

         if idx.from_orbit[0] == idx.to_orbit[0]:
This is the ground state condition

  elif idx.from_orbit[0] != idx.to_orbit[0]:
             if idx.from_spin[0] != idx.to_spin[0]:
This is triplet excited state where alpha spin is flipped to beta
             else:
                 alp.append(idx.to_orbit[0] + h_idx)
                 bet.append(idx.from_orbit[0] + h_idx)
   For the singly excited singlet state

Comment (alexvakimov) [inline src/x_to_libra_qe.py]: okay. I was looking only at the changes - and it didn't appear it was used in the change. I may need to look at the full file.

File: src/spin_indx.py (hunk 1 above)
Comment (Pulokesh) [inline src/spin_indx.py]: Added documentation.

**Pull request merged**